src/controllers/system.ts

```diff
@@ -17,6 +18,7 @@ class SystemController {
         website: true,
         api: true,
         dataStore: _dataStoreStatus,
+        totalAccounts: await Account.getTotalNumber(),
```

Comment:
Might be nice to follow convention here and assign the response of `Account.getTotalNumber()` to a variable (similar to `_dataStoreStatus` on line 15)